db/migrate/20210609081621_add_score_to_profiles.rb

```diff
@@ -0,0 +1,10 @@
+class AddScoreToProfiles < ActiveRecord::Migration[5.2]
+  def up
+    add_column :profiles, :score, :decimal, null: false, default: 0
+    Profile.canonical(false).joins(:test_results).find_each(&:calculate_score!)
```

Comment:
Are you sure the `joins` is a right choice? For a profile with N test results wouldn't it call calculate score N times?

Reply:
My goal was to eliminate profiles without test results but I guess I'm not doing that the right way. I am wondering if a simple `uniq` would fix this :thinking:

Reply:
:see_no_evil: `distinct` :see_no_evil: